docs/slackInvite.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Slack invite
+---
+
+<p>Many bosun devs and users are usually availible on our slack channel. If you have any questions or issues this is the best place to 
+get help. Please fill out this form to automatically receive an invitation to join us:</p>
+
+	<div class="form-group">
+		<label for="email">Email address</label>
+		<input type="email" class="form-control" id="email" placeholder="Enter email">
+	</div>
+	<div class="form-group">
+		<label for="fname">First Name</label>
```

Comment:
Can you show me documentation for that claim? I can't find users.admin.invite documented anywhere. And [this](https://github.com/outsideris/slack-invite-automation) project uses users.admin.invite with just email, no first or last name.